wagtail/models/__init__.py

```diff
@@ -2711,7 +2711,7 @@ def serve_password_required_response(self, request, form, action_url):
 
             if hasattr(settings, "PASSWORD_REQUIRED_TEMPLATE"):
                 warn(
-                    "The `PASSWORD_REQUIRED_TEMPLATE` setting is deprecated - use `WAGTAIL_PASSWORD_REQUIRED_TEMPLATE` instead.",
+                    "The ``PASSWORD_REQUIRED_TEMPLATE`` setting is deprecated - use ``WAGTAIL_PASSWORD_REQUIRED_TEMPLATE`` instead.",
```

Comment:
No need to add double backticks to errors/warnings, this goes straight to stdout/console and is not needing to be formatted.